### DISCUSSION TITLE
Bloated use of the vf-footer, and how can we revitalise a "vf-explore"?

### DISCUSSION THREAD

**Discussion by khawkins98** [visual-framework/vf-core#1529] [Proposals]
The vf-footer in its current usage is becoming a dumping ground for a full sitemap. 

![image](https://user-images.githubusercontent.com/928100/118937902-520b6c00-b94e-11eb-8d2d-ccedfdbd974d.png)

This is not only overwhelming and looks poor, but it is a burden to adopt across all of EMBL's many websites.

(It's also a far cry from the [recommended vf-footer component implementation](https://stable.visual-framework.dev/components/vf-footer/) and doesn't [match our peers](https://www.crick.ac.uk/) well)

We have support from senior stakeholders to thin out the current implementation, but we of course want to ensure content is discoverable. 

Some initial thoughts:

1. We thin out the current EMBL footer to its "most important things" (the missions and contact info?)
2. We create a concept for important but next-level links; we already have a few tools for consideration: 
    - vf-details, vf-tabs (probably not great, but worth consideration)
    - some kind of dropdown/mega menu (see also #1299)
    - Adding "explore this site" as a link or button and formalising the "explore" page concept as a container or page pattern

My preference would be for the final option. 

As @cindyebi is busy with other sprints this week, I may make some initial mockups (or perhaps @sturobson is interested)

**Comment by sturobson** [comment]
I think it should be more concentrated in purpose … 

If you are looking at ideas to make it visibly smaller by using details, tabs, or a new dropdown you're opening up for even more cases of 'put this link in the footer' without feeling the need to question 'why' it needs to be there.

I'd lean more to 

> We thin out the current EMBL footer to its "most important things" (the missions and contact info?)

from your initial thoughts. 

Alas … 6 days to go.

**Comment by khawkins98** [comment]
I have mixed feelings on it, but QUT's use of tabs in the footer is a possible solution to our issue. https://www.qut.edu.au/

![image](https://user-images.githubusercontent.com/928100/123212652-5be03c00-d4c5-11eb-9770-0aaa46f207ef.png)

Also related is the outstanding need for secondary footers #1339

FYSA @cindyebi @nikiforosk as this will come up during the VF sprint on navigation.
